File: mario.py
# bring the get_int from cs50 file
from cs50 import get_int

# no do-while loop so other method
while True:
    h = get_int("Height: ")
    if h < 1 or h > 8:
        h = get_int("Height: ")
    if h >= 1 or h <= 8:
        break
    
# create pyramid
for i in range(h):
    # for spaces
    print((h - 1 - i) * " ", end="")
    # for boxes
    print((i + 1) * "#")
    

File: mario.c
#include <stdio.h>
#include <cs50.h>
//Create Mario pyramid while asking the user for the height 

int main(void)
{
    //intitialize h for height
    int h;
    //make a do-while loop for getting a height value only between 1-8
    do
    {
        h = get_int("What is the pyramid height: ");
    }
    while (h < 1 || h > 8);
    
    //creating rows(x)
    for (int x = 0; x < h; x++)
    {
        //printing out spaces and #'s in these rows

        for (int space = (h - x); space >= 2; space--)
        {
            printf(" ");
               
        }
        //Create hashes for pyramid
        for (int hash = 1; hash <= x + 1; hash++)
        {
            printf("#");
        }
            
        printf("\n");
    }
}
